.github/workflows/ci-components-nightly.yml

```diff
@@ -0,0 +1,39 @@
+name: CI - Components - Nightly
+    
+on:
+  schedule:
+    # Runs at 00:00 UTC
+    - cron:  '0 0 * * *'
+
+env:
+  NODE_VERSION: '16.x'
+
+concurrency:
+   group: ci-components-nightly-${{ github.head_ref || github.ref }}
+   cancel-in-progress: true
+
+jobs:
+  test:
+    name: "A11y Nightly Tests"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - name: Install Node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+          cache-dependency-path: yarn.lock
+      - name: Install Dependencies
+        run: yarn install --immutable
+      - name: Run Tests
+        run: yarn run test:a11y
+        working-directory: packages/components
+      - name: Archive a11y test results
+        uses: actions/upload-artifact@v3
+        with:
+          name: a11y-test-report
+          path: packages/components/test-output/a11y-test-results.json
```

Comment:
@alex-ju I am not sure if it will format to json correctly, do you happen to know? I'll hunt for other examples in the meantime.

Reply:
my understanding of `actions/upload-artifact` is that it takes a local or temporary file and uploads it to GitHub as an artifact of the current flow. what I'm not sure about is how do we get to create `packages/components/test-output/a11y-test-results.json` – running `yarn run test:a11y` only outputs the tests to the terminal, so we might need to update the action to something like this `run: yarn run test:a11y > a11y-test-results.txt`

In regards to the format, I'm wondering if we need to take [the middleware approach](https://github.com/ember-a11y/ember-a11y-testing#test-middleware) to be able to generate it in a json format

Reply:
The qUnit website had some hints, but I've changed it to .txt for now so we can at least see if the job works.